app/errors.js

```diff
@@ -8,11 +8,9 @@ exports.invalidEmail = {
   message: 'Invalid email, it must be a wolox domain.'
 };
 
-exports.databaseError = error => {
-  return {
-    statusCode: 400,
-    message: error.message
-  };
+exports.databaseError = {
+  statusCode: 400,
+  message: 'Error en la BD'
```

Comment:
We have some messages in english and some in spanish we should be consistent and have only 1 language or have multiples configured